src/detection_utils/metrics.py

```diff
@@ -127,25 +127,32 @@ def compute_recall(
     >>> compute_recall(predictions, actual)
     1.0
 
-    # Our highest IoU is 0.6 so let's set our threshold above that
+    Our highest IoU is 0.6 so let's set our threshold above that
+
     >>> compute_recall(predictions, actual, threshold=0.75)
     0.0
     """
     predictions = prediction_detections[:, -1]
     truths = truth_detections[:, -1]
 
     if truths.sum() == 0:
-        return 1  # if there are no targets, then by definition we've found all the targets
+        return (
+            1
+        )  # if there are no targets, then by definition we've found all the targets
```

Comment:
solution from above: put the comment on a line above the return so this isn't weird